controllers/restaurantsChainController.js

```diff
@@ -0,0 +1,102 @@
+import { httpStatus } from '../utils/httpStatusCode.js';
+import { restaurantsChainService } from '../services/restaurantsChainService.js';
+
+const getChains = async function (req, res) {
+  try {
+    const chain = await restaurantsChainService.getChains(req.query);
```

Comment:
your controller should only be responsible for validation, authorisation/authentication not for logic, 
you can consider moving this logic to service layer

Reply:
according to node.js tutorial, controllers handles all the logic behind validating request parameters, query, sending responses with correct codes, so I think this logic is ok (also I took a look at Hania code and she handled it the same way)